pkg/tfvars/ovirt/ovirt.go

```diff
@@ -73,6 +79,21 @@ func TFVars(
 		}
 		cfg.BaseImageLocalFilePath = imageFilePath
 	}
-
+	if len(affinityGroups) > 0 {
+		cfg.AffinityGroups = handleAffinityGroups(affinityGroups, infraID)
+	}
 	return json.MarshalIndent(cfg, "", "  ")
 }
+
+func handleAffinityGroups(ags []*ovirt.AffinityGroup, infraID string) []map[string]interface{} {
+	var agMap []map[string]interface{}
```

Comment:
This is a confusing name for this variable, since it is not a map but is instead a slice.

Reply:
Done